src/Impostor.Api/Config/CheatingHostMode.cs

```diff
@@ -0,0 +1,9 @@
+namespace Impostor.Api.Config
+{
+    public enum CheatingHostMode
+    {
+        Never,
+        IfRequested,
```

Comment:
Some comments would be nice like it was done for `CheatCategory`.

Reply:
Added documentation comments